### PR TITLE
Task 1 part 3 responsive design

### PR DESCRIPTION
**Pull request by ashrafabuodeh**

### CODE REVIEW

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/index.html [hunk 1]
```diff
@@ -1,78 +1,90 @@
...
         <div class="main">
             <div class="hero-image">
-            <img  src="assets/mocha-late-9.png" alt="MOCHA LATE">
-            <div class="hero-text">
-            <p class="product-sans-font">Mocha Late</p>
-            <p class="google-sans-font">Coffee is a brewed drink prepared 
-                from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
-            <pre class="product-sans-font">1 Bag                                            $69.99</pre>
-            </div>
+                <img src="assets/mocha-late-9.png" alt="MOCHA LATE">
```
Comment (araraltawarah98) [inline task_1/index.html]: don't set numbers with the name of the image.

**Review by araraltawarah98 — CHANGES_REQUESTED** [review]

**Review by araraltawarah98 — APPROVED** [review]

**Review by abood454 — APPROVED** [review]

**Review by Mohsen-Awad — APPROVED** [review]

File: task_1/index.html [hunk 2]
```diff
@@ -1,78 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
Comment (mmasaid) [inline task_1/index.html]:
```suggestion
```

File: task_1/index.html [hunk 3]
```diff
@@ -1,78 +1,90 @@
...
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Coffee Style</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
     <link rel="stylesheet" href="./style.css">
```
Comment (mmasaid) [inline task_1/index.html]: No need to use bootstrap
```suggestion
```

File: task_1/index.html [hunk 4]
```diff
@@ -1,78 +1,90 @@
...
-<body>  
+
+<body>
     <div class="wrapper">
         <div class="header">
-                <div class="coffee-bens">
-                <img  src="assets/coffee-beans.png" alt="coffee beans" class="coffee-bens-size" >
+            <div class="coffee-bens">
+                <img src="assets/coffee-beans.png" alt="coffee beans"
+                    class="coffee-bens-size margin-right-with-Burger-menu">
```
Comment (mmasaid) [inline task_1/index.html]:
```suggestion
                    class="coffee-bens-size burger-menu-icon">
```

File: task_1/index.html [hunk 5]
```diff
@@ -1,78 +1,90 @@
...
             </div>
-                <div class="all-links-navbar">
-                    <a class="nav-items menu-item  " onclick="onMenueItemClick('item-home')" id="item-home">Home</a>
-                    <a class="nav-items menu-item "  onclick="onMenueItemClick('item-about-us')" id="item-about-us">About Us</a>
-                    <a class="nav-items menu-item "  onclick="onMenueItemClick('item-our-lands')" id="item-our-lands">Ourlands</a>
-                    <a class="nav-items menu-item "  onclick="onMenueItemClick('item-suppliers')" id="item-suppliers">Suppliers</a>
-                    <a class="nav-items menu-item "  onclick="onMenueItemClick('item-products')" id="item-products">Products</a>
-                    <a class="nav-items menu-item"   onclick="onMenueItemClick('item-contact-us')" id="item-contact-us">Contact Us</a>
+            <img src="./assets/menu.png" alt="icon menue" class="icon-menu" onclick="clickOnBurgerIcon('toggle')"
```
Comment (mmasaid) [inline task_1/index.html]:
```suggestion
            <img src="./assets/menu.png" alt="icon menue" class="icon-menu" onclick="handleBurgerIconClick('toggle')"
```

File: task_1/index.html [hunk 6]
```diff
@@ -1,78 +1,90 @@
...
+                <a class="nav-items menu-item " onclick="onMenueItemClick('item-suppliers')"
+                    id="item-suppliers">Suppliers</a>
+                <a class="nav-items menu-item " onclick="onMenueItemClick('item-products')"
+                    id="item-products">Products</a>
+                <a class="nav-items menu-item" onclick="onMenueItemClick('item-contact-us')"
+                    id="item-contact-us">Contact Us</a>
+            </div>
+            <div class="left-header-images">
+                <img src="assets/shopping-cart.png" alt="shopping cart " class="shopping-cart-size">
+                <img src="assets/profile-image.png" alt="profileImages" class="img-profile">
```
Comment (mmasaid) [inline task_1/index.html]: Be consistent with the alt attribute
```suggestion
                <img src="assets/shopping-cart.png" alt="shopping cart" class="shopping-cart-size">
                <img src="assets/profile-image.png" alt="profile images" class="img-profile">
```

File: task_1/index.js [hunk 7]
```diff
@@ -1,10 +1,15 @@
...
+const onMenueItemClick = (elementId) => {
     const numberOfItem = document.getElementsByClassName('nav-items').length;
-         for(let i = 0 ; i < numberOfItem; i++){
-            if(document.getElementsByClassName('nav-items')[i].classList.contains('active')){
-                document.getElementsByClassName('nav-items')[i].classList.remove('active');
-         }
+    for (let i = 0; i < numberOfItem; i++) {
+        if (document.getElementsByClassName('nav-items')[i].classList.contains('active')) {
+            document.getElementsByClassName('nav-items')[i].classList.remove('active');
         }
```
Comment (mmasaid) [inline task_1/index.js]: Use forEach instead see this example 
```suggestion
      numberOfItem.forEach((el) => {
.....})
```

File: task_1/index.js [hunk 8]
```diff
@@ -1,10 +1,15 @@
...
-        document.getElementById(elementId).classList.add('active');       
-    } 
-    
\ No newline at end of file
+    }
+    document.getElementById(elementId).classList.add('active');
+}
+const clickOnBurgerIcon = (elementId) => {
```
Comment (mmasaid) [inline task_1/index.js]:
```suggestion
const handleBurgerIconClick = (elementId) => {
```

File: task_1/style.css [hunk 9]
```diff
@@ -1,141 +1,213 @@
...
-.coffee-bens{
+.coffee-bens {
   margin: 0 0 0 45px;
 }
+
+.icon-menu {
+  width: 35px;
+  display: none;
+  margin-left: 10px;
+}
```
Comment (mmasaid) [inline task_1/style.css]: Alphabetize Please  
```suggestion
.icon-menu {
  display: none;
  margin-left: 10px;
  width: 35px;
}
```

File: task_1/style.css [hunk 10]
```diff
@@ -1,141 +1,213 @@
...
+
+.display-Burger-icon{
+  visibility: visible;
+}
```
Comment (mmasaid) [inline task_1/style.css]: No need for the default style
```suggestion
```

**Review by mmasaid — CHANGES_REQUESTED** [review]
FR: Please update the style for user profile and the burger icon 
CR: Need to address some comments

**Review by mmasaid — APPROVED** [review]

**Pull request merged**